src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,29 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
+
+import java.util.List;
+
 /**
  * Your implementation of MachineService.
  */
-public class MachineServiceImpl {
+public class MachineServiceImpl implements MachineService{
+    @Override
+    public List<? extends Machine> getAll(List <? extends Machine> machines) {
+        return machines;
+    }
+
+    @Override
+    public void fill(List<? super Machine> machines, Machine value) {
+        machines.add(value);
+
```

Comment:
Logic issue: The `fill()` method should replace the old values in the list with the new ones, not just add a new value to the list. The size of the list passed to `fill()` method should remain the same.